crates/rspack_core/src/chunk.rs

```diff
@@ -223,15 +223,19 @@ impl Chunk {
   }
 
   pub fn get_all_async_chunks(&self, chunk_group_by_ukey: &ChunkGroupByUkey) -> HashSet<ChunkUkey> {
+    use rustc_hash::FxHashSet;
```

Comment:
The file already import `FxHashSet` as `HashSet`

Reply:
I don't want to run CI again. Let's keep it now.